test/e2e/README.md

```diff
@@ -49,22 +55,27 @@ E2E tests can be run from the Velero repository root by running `make test-e2e`.
 Below is a mapping between `make` variables to E2E configuration flags.
 1. `CREDS_FILE`: `-credentials-file`. Required.
 1. `BSL_BUCKET`: `-bucket`. Required.
-1. `PLUGIN_PROVIDER`: `-plugin-provider`. Required.
+1. `CLOUD_PROVIDER`: `-cloud-provider`. Required
+1. `OBJECT_STORE_PROVIDER`: `-object-store-provider`. Required when kind is the cloud provider.
 1. `VELERO_CLI`: the `-velerocli`. Optional.
 1. `VELERO_IMAGE`: the `-velero-image`. Optional.
 1. `BSL_PREFIX`: `-prefix`. Optional.
 1. `BSL_CONFIG`: `-bsl-config`. Optional.
 1. `VSL_CONFIG`: `-vsl-config`. Optional.
 
-For example, E2E tests can be run from Velero repository roots using the below command:
+For example, E2E tests can be run from Velero repository roots using the commands below:
 
-1. Run Velero tests using AWS as the storage provider:
+1. Run Velero tests in a kind cluster with AWS (or Minio) as the storage provider:
     ```bash
-    BSL_PREFIX=<PREFIX_UNDER_BUCKET> BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds PLUGIN_PROVIDER=aws make test-e2e
+    BSL_PREFIX=<PREFIX_UNDER_BUCKET> BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds CLOUD_PROVIDER=kind OBJECT_STORE_PROVIDER=aws make test-e2e
     ```
-1. Run Velero tests using Microsoft Azure as the storage provider:
+1. Run Velero tests in an AWS cluster:
     ```bash
-    BSL_CONFIG="resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID,subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID" BSL_BUCKET=velero CREDS_FILE=~/bin/velero-dev/aks-creds PLUGIN_PROVIDER=azure make test-e2e
+    BSL_PREFIX=<PREFIX_UNDER_BUCKET> BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds CLOUD_PROVIDER=aws make test-e2e
+    ```
+1. Run Velero tests in a Microsoft Azure cluster:
+    ```bash
+    BSL_CONFIG="resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID,subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID" BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds CLOUD_PROVIDER=azure make test-e2e
```

Comment:
```suggestion
BSL_CONFIG="resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID,subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID" BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/azure-creds CLOUD_PROVIDER=azure make test-e2e
```

Reply:
I updated this, no more README changes please.